### PR TITLE
Updated raffo.ttl with groupings

### PR DESCRIPTION
**Pull request by Pastitas**
Added lv2 groups to make it more compatible with zynthian.

### CODE REVIEW

File: raffo.ttl [hunk 1]
```diff
@@ -45,10 +48,12 @@
 	a lv2:ControlPort, lv2:InputPort;
 	lv2:index 2;
 	lv2:symbol "volume";
+
```
Comment (nicoroulet) [inline raffo.ttl]: Please remove newline

**Review by nicoroulet — CHANGES_REQUESTED** [review]
Thanks!

File: raffo.ttl (hunk 1 above)
Comment (Pastitas) [reply raffo.ttl]: fixed on  d07bbcf

**Comment by julianpalladino** [comment]
Thanks for the contribution @Pastitas!

**Comment by Pastitas** [comment]
You're welcome! Thanks for the great synth!

**Pull request merged**